ui/main/src/app/modules/logging/logging.component.ts

```diff
@@ -125,39 +117,10 @@ export class LoggingComponent implements OnDestroy, OnInit {
         this.resultsNumber = 0;
     }
 
-    private displayMessage(i18nKey: string, msg: string, severity: MessageLevel = MessageLevel.ERROR) {
-        this.store.dispatch(
-            new AlertMessageAction({alertMessage: {message: msg, level: severity, i18n: {key: i18nKey}}})
-        );
-    }
-
     sendQuery(page_number): void {
         this.technicalError = false;
-        const publishStart = this.extractTime(this.loggingForm.get('publishDateFrom'));
-        const publishEnd = this.extractTime(this.loggingForm.get('publishDateTo'));
-
-        if (
-            publishStart != null &&
-            !isNaN(publishStart) &&
-            publishEnd != null &&
-            !isNaN(publishEnd) &&
-            publishStart > publishEnd
-        ) {
-            this.displayMessage('shared.filters.publishEndDateBeforeStartDate', '', MessageLevel.ERROR);
-            return;
-        }
 
-        const activeStart = this.extractTime(this.loggingForm.get('activeFrom'));
-        const activeEnd = this.extractTime(this.loggingForm.get('activeTo'));
-
-        if (
-            activeStart != null &&
-            !isNaN(activeStart) &&
-            activeEnd != null &&
-            !isNaN(activeEnd) &&
-            activeStart > activeEnd
-        ) {
-            this.displayMessage('shared.filters.activeEndDateBeforeStartDate', '', MessageLevel.ERROR);
+        if (!this.filtersTemplate.isFormValid()) {
```

Comment:
move this control in ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.ts in method quey() , you do not emit the search if invalid